api/resource.rb

```diff
@@ -42,6 +42,12 @@ module Properties
       # GET/DELETE requests only.  In particular, this is often used
       # to add query parameters.
       attr_reader :self_link_query
+      # URL to be used to retrieve a list of these resources (for InSpec)
+      attr_reader :list_url
+      # This is the type of response from the list URL. It contains
+      # the name of the list of items within the json, as well as the
+      # type that this list should be
+      attr_reader :list_url_response
```

Comment:
I'd call it list_url_type, or give an example.

Reply:
Oh, I see now that what I really want to ask for is to include 'this is of type Api::Resource::ResponseList' in the comment.